contracts/projectRegistry/ProjectRegistry.sol

```diff
@@ -52,6 +54,7 @@ contract ProjectRegistry is Initializable {
 
     event ProjectCreated(uint256 indexed projectID, address indexed owner);
     event MetadataUpdated(uint256 indexed projectID, MetaPtr metaPtr);
+    event ProgramMetadataUpdated(uint256 indexed projectID, MetaPtr programMetadata);
```

Comment:
curious if we can reuse `MetadataUpdated`

Reply:
I don't see why not? It's just the name of the emitted event property that is different 🤔

Reply:
Yeah it becomes confusing. 
We'd have to emit another argument which says type as project-metadata or program-metadata. That would make it easier for clients (but still painful)